Lab 4

d3nik opened the pull request.

d3nik committed:
Patch 4.1

– svg sprites added in folder images\icons;
– added icons in header, main and footer.

d3nik committed:
Patch 4.2

– icons color fixed;
– intro background image added;
– :hover and :focus icons color changed (where needed);
– added permanent shadows to team cards;
– added :hover and :focus shadows to filter buttons and portfolio cards;

d3nik committed:
Patch 3.2.1 (hotfix)

– Fixed intro background image size;

d3nik commented:
Ім'я останнього коміта Patch 4.2.1 (hotfix)" 
замість "patch 3.2.1 (hotfix)"

vozniuk-lntu reviewed (approved):
1. пофіксь контейнер у герої, вирівняй іконки відносно тексту вертикально по центру
2. картинка в карточці портфоліо вилазить за карточку
3. footer-right-side заголовок капсом і там верхній маржин то костиль

https://docs.google.com/document/d/1bv50jpMgYkjKHjibr75pwwGWuXgCZEQC/edit?usp=sharing&ouid=104518963218417128491&rtpof=true&sd=true

d3nik committed:
Patch 4.2.2 (hotfix)

– icons in header vertically aligned;
– fixed padding in some containers;
– fixed images in portfolio cards;
– changed footer-right side h3 to uppercase and replaced top margin to top padding.

d3nik committed:
Merge branch 'lab-4' of https://github.com/d3nik/Serediuk-Denys-IPZ-34 into lab-4

d3nik committed:
Patch 4.3.3 (hotfix)

– fixed intro image position.
